devdoc/amqp_management_requirements.md

```diff
@@ -180,17 +180,17 @@ int amqp_management_execute_operation_async(AMQP_MANAGEMENT_HANDLE amqp_manageme
 
 **SRS_AMQP_MANAGEMENT_01_055: [** `amqp_management_execute_operation_async` shall start an AMQP management operation. **]**
 
-**SRS_AMQP_MANAGEMENT_01_056: [** On success it shall return 0. **]**
+**SRS_AMQP_MANAGEMENT_01_056: [** On success it shall return an ASYNC_OPERATION_HANDLE. **]**
```

Comment:
>l return an ASYNC_OPERATION_HANDLE. **]** [](start = 52, length = 41)

I believe we should spec what function goes into the async operation so that we can actually spec what that does. As it is now the specs don't say what should happen when the cancel handler is triggered.

Reply:
Same for the cbs ones.

---
In reply to: [607365459](https://github.com/Azure/azure-uamqp-c/pull/387#discussion_r607365459) [](ancestors = 607365459)

Reply:
Spec added.